unit_tests/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+set(ExtractorTest_SOURCES
+	extractor_tests.cpp
+	extractor/compressed_edge_container.cpp
+	extractor/graph_compressor.cpp
+	extractor/raster_source.cpp)
+
+set(EngineTest_SOURCES
+	engine_tests.cpp
+	engine/douglas_peucker.cpp
+	engine/geometry_string.cpp)
+
+set(UtilTest_SOURCES
+	util_tests.cpp
+	util/bearing.cpp
+	util/binary_heap.cpp
+	util/coordinate.cpp
+	util/duration_parsing.cpp
+	util/dynamic_graph.cpp
+	util/io.cpp
+	util/range_table.cpp
+	util/static_graph.cpp
+	util/static_rtree.cpp
+	util/string_util.cpp)
```

Comment:
In my opinion you should know _exactly_ what files your building an object file with. That's also how we did it in rtd. In the worst case you get a linker error. Anyway, it's not worth it bike-shedding on smth. like this. Will re-work with globs.